.github/workflows/build.yml

```diff
@@ -0,0 +1,359 @@
+---
+name: build
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+  schedule:
+    - cron: '0 10 * * *'  # everyday at 10am
+  repository_dispatch:
+    # Respond to rebuild requests. See: https://github.com/cisagov/action-apb/
+    types: [apb]
+  workflow_dispatch:
+    inputs:
+      remote-shell:
+        description: "Debug with remote shell"
+        required: true
+        default: false
+      image-tag:
+        description: "Tag to apply to pushed images"
+        required: true
+        default: dispatch
+
+env:
+  BUILDX_CACHE_DIR: ~/.cache/buildx
+  IMAGE_NAME: cisagov/domain-manager-api
+  PIP_CACHE_DIR: ~/.cache/pip
+  # Disabled following platforms due to issues with GO
+  # linux/arm/v6,linux/ppc64le,linux/s390x
```

Comment:
Let me know if this is better https://github.com/cisagov/domain-manager-api/pull/194/commits/8074b12a30685c8a5450cc8c109824df16e86ebf

linux/s390x and linux/ppc64le never actually failed for me, but I started them building over 2 hours ago and never succeeded. As these platforms are older platforms I though it would be fine to disable them to not slow down our development.

Reply:
That's fine for now- thanks.  I'd love to have a bit more understanding about why those builds are taking so long, but I understand that you can't spend all of your time debugging those other platforms.